cmake/schemes/url_sha1_autotools.cmake.in

```diff
@@ -58,3 +58,11 @@ hunter_autotools_project("@HUNTER_EP_NAME@"
     BOOTSTRAP
       "${BOOTSTRAP}"
 )
+
+foreach(PKG_CONFIG_MODULE ${PKGCONFIG_EXPORT_TARGETS})
+  configure_file(
+      "@HUNTER_GLOBAL_SCRIPT_DIR@/pkgconfig-export-targets.cmake.in"
+      "@HUNTER_PACKAGE_INSTALL_PREFIX@/cmake/Hunter/Find${PKG_CONFIG_MODULE}.cmake"
```

Comment:
Ok, I can put them into `${PKG_CONFIG_MODULE}Config.cmake` so it is less code into `hunter_finalize`